ext/MakieExt/plot.jl

```diff
@@ -1,147 +1,172 @@
-using GLMakie: Point2f0
+using Makie: Point2f0
 
-function GLMakie.convert_arguments(::PointBased, contres::AbstractBranchResult, vars = nothing, applytoY = identity, applytoX = identity)
+function Makie.convert_arguments(::PointBased, contres::AbstractBranchResult, vars=nothing, applytoY=identity, applytoX=identity)
```

Comment:
My coding style is to have spaces around =

Reply:
Ah yes, sorry about that, I hit auto indent and forgot to check it. I just don't know yet in vscode how to change the formatting settings. Would you know ?

Reply:
not really